src/providers/aws.rs

```diff
@@ -287,12 +293,14 @@ where
 /// such instances must be declared in advance (1-6 hours). By default, we use 6 hours (the
 /// maximum). To change this, RegionLauncher respects the limit specified in
 /// [`Launcher::set_max_instance_duration`](Launcher::set_max_instance_duration).
-#[derive(Default)]
+#[derive(Educe, Default)]
+#[educe(Debug)]
 pub struct RegionLauncher {
     pub region: rusoto_core::region::Region,
     security_group_id: String,
     ssh_key_name: String,
     private_key_path: Option<tempfile::NamedTempFile>,
+    #[educe(Debug(ignore))]
```

Comment:
Is `Ec2Client` really not `Debug`?!

Reply:
https://docs.rs/rusoto_ec2/0.41.0/rusoto_ec2/struct.Ec2Client.html

nope :(